dlio_benchmark/configs/workload/llama_70b_zero3.yaml

```diff
@@ -0,0 +1,46 @@
+# we mimic the checkpoint data for megatron-deepspeed
+model: 
+  name: llama_70b
+  type: transformer
+  model_size: 30102
+  num_layers: 80
+  parallelism: 
+    tensor: 8
+    pipeline: 1
+    zero_stage: 3
+  transformer: 
+    vocab_size: 128000
+    hidden_size: 8192
+    ffn_hidden_size: 28672
+
+framework: pytorch
+
+workflow:
+  generate_data: True
+  train: True
+  checkpoint: True
+
+dataset: 
+  data_folder: data/llama_70b/
+  format: mmap_indexed_binary
+  num_files_train: 1
+  num_samples_per_file: 1048576
+  record_length: 2048
+  
+reader: 
+  data_loader: pytorch
+  batch_size: 16
+  read_threads: 1
+  file_shuffle: seed
+  sample_shuffle: seed
+
+train:
+  epochs: 1
+  computation_time: 5 # 2.44 sec per step
+  total_training_steps: 5
+
+checkpoint:
+  checkpoint_folder: checkpoints/llama_70b
+  steps_between_checkpoints: 1
+  model_datatype: fp16
+  optimizer_datatype: fp32
```

Comment:
I feel these should be in model. and you missed adding a new line at end of file here.

Reply:
These were not fixed.